bpf/lib/encap.h

```diff
@@ -184,7 +184,7 @@ encap_and_redirect_with_nodeid(struct __ctx_buff *ctx, __u32 tunnel_endpoint,
 static __always_inline int
 encap_and_redirect_lxc(struct __ctx_buff *ctx, __u32 tunnel_endpoint,
 		       __u8 encrypt_key __maybe_unused,
-		       struct endpoint_key *key, __u32 seclabel, __u32 monitor)
+			struct endpoint_key *key, __u32 seclabel, __u32 monitor)
```

Comment:
nit: These whitespace changes seem to be spurious. Please consider dropping them, or if they're fixing the style in the tree, submitting as a separate commit (or even PR).

Reply:
For what it's worth, I looked at this one in the tree and if you have the tabs set up kernel-style, the indents are correct. So these changes are unnecessary.

Reply:
the PR has been re-written multiple times and I may messed up the original format, I will start from clean original file.